.github/workflows/cd.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   dist:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
```

Comment:
FYI, you need to set the fetch-depth for v2+, otherwise you’ll produce 0.0.0 every time (which sadly will “work” once).

Reply:
It’s set in the other action:
```yaml
      with:
         fetch-depth: 0
```

Reply:
I will create a follow-up MR ...

Reply:
Done with https://github.com/scikit-hep/hepunits/pull/66.